src/tests/stream-tcp-list.c

```diff
@@ -572,15 +572,15 @@ static int StreamTcpReassembleTest02(void)
     OverlapVISTABefore(UINT_MAX - 10);
     OverlapVISTASame(UINT_MAX - 10);
     OverlapVISTAAfter(UINT_MAX - 10);
-    return 1;
+    PASS;
 }
 
 
 /** \test Linux policy
  */
 static int StreamTcpReassembleTest03(void)
 {
-    OverlapLINUX(0);
+   FAIL_IF(OverlapLINUX(0) ==0);
```

Comment:
This won't matter for this specific work, as this change isn't needed. But, when you add code, please ensure that it is aligned with pre-existing code - which you can see isn't here, causing a formatting check CI failure ;)

Reply:
Im sorry. I dont follow

Reply:
Which part, exactly?
What I mean is that, if you compare the start of this line that you've added to the start the the lines below, you'll notice that they're not aligned. They must be, otherwise the `clang-format` check will fail, as you can see here: https://github.com/OISF/suricata/actions/runs/6667576626/job/18129597039